### ISSUE TITLE
NewMountPointFromLine fails under WSL

### ISSUE THREAD

**Issue by stevenh** [uber-go/automaxprocs#72]
If you use `NewMountPointFromLine` on WSL it will fail due to the spacing in paths in super options.

```
   --- FAIL: TestLogger/default (0.00s)
        uber-go/automaxprocs/maxprocs/maxprocs_test.go:69: 
            	Error Trace:	maxprocs_test.go:69
            	Error:      	Received unexpected error:
            	            	invalid format for MountPoint: "560 77 0:138 / /Docker/host rw,noatime - 9p drvfs rw,dirsync,aname=drvfs;path=C:\\Program Files\\Docker\\Docker\\resources;symlinkroot=/mnt/,mmap,access=client,msize=262144,trans=virtio"
            	Test:       	TestLogger/default
            	Messages:   	Set failed
```

**Comment by hhromic** [comment]
@sywhang this was a duplicate of #65. Given that release `v1.5.3` fixes the issue, issue #65 should be closed too.
Btw, thanks for releasing a fix for this!

**Comment by sywhang** [comment]
@hhromic thanks, closed #65.
